增加中文维基百科用户贡献数统计

scvoet opened the pull request.

spencerwooo commented:
Thanks! I'll look into this when I have time later this week.

spencerwooo reviewed (approved):
👍

spencerwooo commented on src/utils/handlers/wikipedia-zh.js:
```diff
@@ -0,0 +1,41 @@
+      failedMsg: '',
+    }
+  
+    if (response.stats.query.users.0.missing = '') {
```
This doesn't seem right.

spencerwooo commented on src/utils/handlers/wikipedia-zh.js:
```diff
@@ -0,0 +1,41 @@
+      res.failedMsg = 'Wikipedia-zh user not found'
+    } else {
+      // Wikipedia-zh user found
+      res.subs = stats.query.users.0.editcount
```
Neither does this.

spencerwooo reviewed (commented):
Sorry I was in a rush in merging PRs today. Could you give me an example for a valid Wikipedia username that I can test with the API you provided here?

```
https://zh.wikipedia.org/w/api.php?action=query&list=users&ususers=${name}&usprop=editcount&format=json
```

Just a username I can substitute `${name}` with.

scvoet commented:
> 
> 
> Sorry I was in a rush in merging PRs today. Could you give me an example for a valid Wikipedia username that I can test with the API you provided here?
> 
> ```
> https://zh.wikipedia.org/w/api.php?action=query&list=users&ususers=${name}&usprop=editcount&format=json
> ```
> 
> Just a username I can substitute `${name}` with.

Sure, and `scvoet` is available.

spencerwooo commented:
@scvoet I corrected and cleaned up the code for you on PR #25 if you would like to have a look.

In the meantime, could you provide a little bit more for the documentation part of Substats - Wikipedia (zh)? Which is at [API details - Wikipedia (zh)](https://substats.spencerwoo.com/api.html#wikipedia-zh). The `queryKey` part is especially missing key information as I myself is not familiar with the API.

scvoet commented:
> 
> 
> @scvoet I corrected and cleaned up the code for you on PR #25 if you would like to have a look.
> 
> In the meantime, could you provide a little bit more for the documentation part of Substats - Wikipedia (zh)? Which is at [API details - Wikipedia (zh)](https://substats.spencerwoo.com/api.html#wikipedia-zh). The `queryKey` part is especially missing key information as I myself is not familiar with the API.

Of course, I will finish the document today.
